prac_07/guitar.py

```diff
@@ -0,0 +1,27 @@
+YEAR = 2022
+VINTAGE_YEAR = 50
+
+
+class Guitar:
+    """a guitar object"""
+
+    def __init__(self, name='', year=0, cost=0):
+        self.name = name
+        self.year = int(year)
+        self.cost = float(cost)
```

Comment:
Instead of explicitly making cost a float as you have done, either have "cost=0.0" or "cost: float" in the default arguments.